data/data/azure/bootstrap/main.tf

```diff
@@ -188,7 +188,7 @@ resource "azurerm_linux_virtual_machine" "bootstrap" {
   location              = var.azure_region
   resource_group_name   = var.resource_group_name
   network_interface_ids = [azurerm_network_interface.bootstrap.id]
-  size                  = var.azure_bootstrap_vm_type
+  size                  = var.azure_master_vm_type
```

Comment:
We also need to do something about the type for the os_disk. If the user selects a VM type that does not support premium IO, then the install will fail to create the bootstrap VM. Consequently, we need to use the same disk type for the bootstrap VM as we do for the master VMs.